<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleEarthOptionsDialog</class>
 <widget class="QDialog" name="GoogleEarthOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Google Earth options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="View_groupBox">
     <property name="title">
      <string>View</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QCheckBox" name="showDescription_checkBox">
        <property name="text">
         <string>show description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showEventLabel_checkBox">
        <property name="text">
         <string>show event label</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="Marker_groupBox">
     <property name="title">
      <string>Marker</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QComboBox" name="IconColor_comboBox">
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="IconSize_spinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Symbol:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="IconSymbol_comboBox">
        <item>
         <property name="text">
          <string>circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>star</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>triangle</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="OutputFilename_groupBox">
     <property name="title">
      <string>Output file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="GoogleEarthFilename_lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="BrowseGoogleEarthFilename_pushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="GoogleEarth_groupBox">
     <property name="title">
      <string>Google Earth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="GoogleEarthProgram_lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="BrowseGoogleEarthProgram_pushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="StartGoogleEarth_checkBox">
        <property name="text">
         <string>start Google Earth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showDescription_checkBox</tabstop>
  <tabstop>IconColor_comboBox</tabstop>
  <tabstop>IconSize_spinBox</tabstop>
  <tabstop>GoogleEarthFilename_lineEdit</tabstop>
  <tabstop>BrowseGoogleEarthFilename_pushButton</tabstop>
  <tabstop>GoogleEarthProgram_lineEdit</tabstop>
  <tabstop>BrowseGoogleEarthProgram_pushButton</tabstop>
  <tabstop>StartGoogleEarth_checkBox</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
